static/src/javascripts/projects/common/modules/experiments/tests/next-in-series.js

```diff
@@ -0,0 +1,71 @@
+define([
+    'common/utils/$',
+    'lodash/collections/find'
+], function (
+    $,
+    find
+) {
+    var path = '/surveys/404-test/next-in-series/';
+    var render = function (state) {
+        return '<div class="next-in-series-test">' +
+            '<h3>Coming up next week</h3>' +
+            '<h4>' + state.title + '</h4>' +
+            '<p class="next-in-series-test__teaser">' + state.trail + '</p>' +
+            '<a href="' + path + state.id + '"' +
+                ' data-link-name="next in series | remind me"' +
+                ' class="button button--large next-in-series-test__remind-me-link">Remind me</a>' +
+        '</div>';
+    };
+
+    var $articleBody = $('.content__article-body');
+
+    var allSeries = [
+        {
+            id: 'experience',
+            pageId: 'lifeandstyle/2016/mar/25/i-paid-to-have-my-daughter-kidnapped-experience',
+            title: 'I was attacked by a seal',
+            trail: '‘It rolled on top of me, pushing me under water. I was trapped under a mountain of blubber and muscle…’'
```

Comment:
Agreed, i think this line is the dominant factor in whether the test is successful or not! :swimmer: